src/_skytable.ts

```diff
@@ -171,6 +171,18 @@ export class Skytable {
     }
   }
 
+  async mset(...args: string[]): Promise<Integer> {
```

Comment:
Thank you!

Can you separate `mset` from this PR and keep it `mget` only?

You can open a new PR for `mset` if you wish.

Ideally `mset` should accept argument at least from types `Record<string, string>` and `Map<string, string>`.

Reply:
I have removed `mset` from this PR and will raise new one with suggested changes.